### ISSUE TITLE
Update the look of the FilterBar

### ISSUE THREAD

**Issue by jack-brinkman** [gbif/hosted-portals#248]

**Comment by MortenHofft** [comment]
https://github.com/gbif/gbif-web/blob/8edd7800000259de4ecf6166906bd4a16b7147f0/packages/react-components/locales/translations/en/components/search.json#L16

This looks like a mistake. It will not update the filter button text, but it will change text other places.

**Comment by jack-brinkman** [comment]
That's all right - if you're not too keen on it being a part of the hosted portals, I'll leave it for now. With the funnel icon, it did look a tad _odd_ anyway. I'll revisit this issue if it's brought up again.
